DemoServer/AdditionalLanguages/nodejs/demo/spatial/spatialQuery/index.js

```diff
@@ -1,24 +1,40 @@
 const { documentStore } = require('../../common/docStoreHolder');
-const { PointField } = require('ravendb');
 const { Employee } = require('../../common/models');
+//region Usings
+const { PointField } = require('ravendb');
+//endregion
 
 async function run ({ radius }) {
     radius = radius != null ? radius : 2;
 
+    //region Demo
     const session = documentStore.openSession();
 
+    //region Step_1
     const centerPointLng = -122.3150148;
     const centerPointLat = 47.63016419999999;
 
     const wktCircle = 'CIRCLE(' + centerPointLng + ' ' + centerPointLat + ' d=' + radius + ')';
+    //endregion
 
+    //region Step_2
     const employeesWithinCircle = await session.query(Employee)
         .spatial(
+    //endregion
+            //region Step_3
             new PointField('Address.Location.Latitude', 'Address.Location.Longitude'),
+            //endregion
+            //region Step_4
             spatialCriteria => spatialCriteria.relatesToShape(wktCircle, 'Within', 'Miles', 0)
```

Comment:
@ml054 
The C# client has a default value for distErrorPercent `double distErrorPercent = 0.025`
 but here (and in Java client as well)
 we pass `0` for that param as no default value is defined in method signature.
 
 Can that be added ?

Reply:
just remove this param and match c# version

Reply:
can't just remove the `0` from the Java demo
no matching overload method for that